.github/workflows/marldown-links.yml

```diff
@@ -0,0 +1,25 @@
+name: Check Markdown links
+
+on: [push,  pull_request]
```

Comment:
I would use only `pull_request`, we don't want to see a _red_ when things are merged to `master`

Reply:
My goal was allow it to run when we work in forks branches.

I'll make sure it wont be red here